### ISSUE TITLE
[feature] disable third-party Move module (smart contract) publishing

### ISSUE THREAD

**Issue by 0xzoz** [0LNetworkCommunity/libra-framework#149]
**Is your feature request related to a problem? Please describe.**

TL;DR  for new comers: the Open Libra  product is not a compute platform but utilizes advanced state transition technology for the core product: the Libra Coin. The `libra-framework` code serves for a rich programming experience for Libra, and not a generalized compute platform (more information will come on this). The ability to facilitate a platform that allows smart contracts is not in the scope of the product (as of now). 

The experimental smart contract publishing ability should be removed from L1, as we concentrate on network stability and our core product. If compute becomes a compelling sub-product the strategy is to be more like Bitcoin less like Ethereum; generalized compute should be pushed to L2, for safety and economic soundness of L1 and native asset.

**Describe the solution you'd like**

Disable the ability to publish smart contracts modules on arbitrary addresses. Does not affect Move transaction scripts, nor 0x1 address module usage by third parties.

**Describe alternatives you've considered**

N/A

**Additional context**

Thus far the feature was experimental, and disabling does not have any impact on historical, current, or planned ecosystem participants.

**Comment by heavysword1** [comment]
Is there a list of prebuilt scripts

**Comment by 0xzoz** [comment]
Closing due to being merged into main in #170
